sunpy/io/special/asdf/tags/tests/test_genericmap.py

```diff
@@ -1,6 +1,7 @@
 """
 isort:skip_file.
 """
+from sunpy.tests.helpers import asdf_entry_points
```

Comment:
wtaf

Reply:
Not sure why it seems to isort.

Reply:
sunpy/io/special/asdf/tags/tests/test_coordinate_frames.py:15:1: E402 module level import not at top of file
sunpy/io/special/asdf/tags/tests/test_genericmap.py:16:1: E402 module level import not at top of file

thats why.